lib/Suggestions.js

```diff
@@ -29,12 +50,12 @@ class Suggestions extends Component {
     const { props } = this;
     const suggestions = props.suggestions.map(function (item, i) {
       return (
-        <li key={i}
-        onMouseDown={props.handleClick.bind(null, i) }
-        onMouseOver={props.handleHover.bind(null, i) }
-        className={i == props.selectedIndex ? "active" : ""}>
-        <span dangerouslySetInnerHTML={this.markIt(item, props.query) } />
-        </li>
+          <li key={i}
```

Comment:
I think changes to this code might be out of scope. Looks like my ide updated the indentation here, but that wasn't intentional.